spec/controllers/email_alert_subscriptions_controller_spec.rb

```diff
@@ -91,21 +91,15 @@
         stub_content_store_has_item("/cma-cases/email-signup", cma_cases_signup_content_item)
       end
 
-      context "when no filters are provided" do
-        it "redirects to the subscription url" do
-          email_alert_api_has_subscriber_list(
-            "tags" => {
-              "format" => { any: %w[cma_case] },
-            },
-            "subscription_url" => "http://www.gov.uk/default-subscription-to-cma-cases",
-          )
-
+      context "when a required filter is not provided" do
+        it "returns the user to the signup page with an error" do
           post :create, params: { slug: "cma-cases" }
-          expect(subject).to redirect_to("http://www.gov.uk/default-subscription-to-cma-cases")
+          expect(response).to be_successful
+          expect(response).to render_template("new")
```

Comment:
Is it right that this successfully renders the `new` page, rather than redirecting?

Reply:
Yeah, now that CMA cases has a facet that requires a value (I changed the fixture in finder-frontend to reflect the change [here](https://github.com/alphagov/specialist-publisher/pull/1564/files#diff-63cd0912e8dee64396223c25801cccd4R30)), we want to ensure a user that doesn't set it will render `new` rather than creating a subscriber list.